Source/Icebreaker/Cards/PairUpNotificationCard.cs

```diff
@@ -22,30 +22,30 @@ public class PairUpNotificationCard
         /// </summary>
         /// <returns>An attachment to append to a message.</returns>
         /// <param name="teamName">The team name.</param>
-        /// <param name="firstPerson">The first person in the pair up.</param>
-        /// <param name="secondPerson">The second person in the pair up.</param>
+        /// <param name="sender">The first person in the pair up.</param>
+        /// <param name="recipient">The second person in the pair up.</param>
         /// <param name="botDisplayName">The bot display name.</param>
         public static Attachment GetCard(
             string teamName,
-            ChannelAccount firstPerson,
-            ChannelAccount secondPerson,
+            ChannelAccount sender,
+            ChannelAccount recipient,
             string botDisplayName)
         {
-            var person1 = firstPerson.Properties.ToObject<TeamsChannelAccount>();
-            var person2 = secondPerson.Properties.ToObject<TeamsChannelAccount>();
-
-            var firstPersonFirstName = string.IsNullOrEmpty(person1.GivenName) ? firstPerson.Name : person1.GivenName;
-            var secondPersonFirstName = string.IsNullOrEmpty(person2.GivenName) ? secondPerson.Name : person2.GivenName;
-            var title = string.Format(Resources.MeetupTitle, firstPersonFirstName, secondPersonFirstName);
+            var senderGivenName = string.IsNullOrEmpty(sender.Properties.ToObject<TeamsChannelAccount>().GivenName) ?
```

Comment:
The right way to convert ChannelAccount->TeamsChannelAccount is here:
```
JObject.FromObject(channelAccount).ToObject<TeamsChannelAccount>()
```

from
https://github.com/OfficeDev/BotBuilder-MicrosoftTeams/blob/7e078a611d727d27c1df81b33df9823e6b92ae64/CSharp/Library/Microsoft.Bot.Connector.Teams.Shared/ConversationExtensions.cs#L74

Reply:
done with this